setup.py

```diff
@@ -48,8 +48,8 @@ def get_version_file_path():
         "MarkupSafe==3.0.2",
         "mergedeep==1.3.4",
         "packaging==24.2",
-        "python-gitlab==5.3.1",
-        "python-gitlab[graphql]==5.3.1",
+        "python-gitlab==5.6.0",
+        "python-gitlab[graphql]==5.6.0",
```

Comment:
I believe these updates are already done in `main` branch. Let's undo this change.

Reply:
Done, merged main into my branch